conformance/utils/suite/features.go

```diff
@@ -61,7 +61,7 @@ const (
 var GatewayExtendedFeatures = sets.New(
 	SupportGatewayPort8080,
 	SupportGatewayStaticAddresses,
-).Insert(GatewayCoreFeatures.UnsortedList()...)
```

Comment:
I'd always interpreted "Extended" here as everything that is "Extended" or greater, essentially "Extended" + "Core". I'm really hesitant to change that here even if no one on GitHub is currently using this variable. I'd rather just more clearly document the current meaning.

Reply:
Looks like `HTTPRouteExtendedFeatures` does not include the core feature list, so whichever way we go, would be good to standardize on one approach

Reply:
Good find @sunjayBhatia! I agree that we should be consistent across our extended feature lists here, so I take back my objection. I think Lior's idea to keep "Core" features out of "Extended" lists probably is the most sensible option here, we just should probably add comments to clearly state that distinction.

Reply:
yep that was the intention of the PR, to be consistent. 

So LGTM ? 🙂